src/gamecontroller.cs

```diff
@@ -121,6 +121,8 @@ partial interface GCControllerDirectionPad {
 
 	[iOS (7,0)]
 	[Mac (10,9, onlyOn64: true)]
+	[Deprecated (PlatformName.MacOSX, 10, 12)]
+	[Deprecated (PlatformName.iOS, 10, 0)]
```

Comment:
You'll need a tvOS entry too - it's implied as always available in tvOS (by `[iOS (7,0)]`) but that does not apply to deprecation

Reply:
Still strange that this happens only in some, not all bots :|